libdevcrypto/CMakeLists.txt

```diff
@@ -24,10 +24,8 @@ target_link_libraries(${EXECUTABLE} ${DB_LIBRARIES})
 target_link_libraries(${EXECUTABLE} ${CRYPTOPP_LIBRARIES})
 target_link_libraries(${EXECUTABLE} scrypt)
 target_link_libraries(${EXECUTABLE} devcore)
-if (NOT WIN32)
-	add_definitions(-DETH_HAVE_SECP256K1)
-	target_link_libraries(${EXECUTABLE} secp256k1)
-endif ()
+add_definitions(-DETH_HAVE_SECP256K1)
```

Comment:
It's possible this may not work when we upgrade to bitcoin-core; openssl and/or different compile flags may be required.